inst/PlotsFileEditor/app.R

```diff
@@ -0,0 +1,310 @@
+#---------- Initialization of app ----------#
+# Load required packages
+library(shiny)
+
+#---------- User interface ----------#
+ui <- fluidPage(
+  tags$style(type = "text/css", ".modal-dialog {overflow: auto; width: auto; background:white;}"),
+  shinyjs::useShinyjs(),
+  headerPanel("PlotsFile Editor"),
+  br(),
+  actionButton("changeProjConfig", "Change project configuration file"),
```

Comment:
Use `fluidRow` and `column` instead. 
+ I would put the textOutput **next** to the changeProjConfig button.
+ add text explaining what the app does.